ui/index.d.ts

```diff
@@ -471,7 +479,7 @@ export declare abstract class AzureTreeItem<TRoot extends ISubscriptionContext =
     /**
      * Contains subscription information specific to the root of this branch of the tree.
      */
-    public readonly root: TRoot;
```

Comment:
Having only a get method should behave as readonly.  Also, the implemtation was already using a get accessor: https://github.com/microsoft/vscode-azuretools/blob/main/ui/src/tree/AzureTreeItem.ts#L15